tests/snmp/test_snmp_link_local.py

```diff
@@ -4,7 +4,7 @@
 from tests.common.utilities import wait_until
 
 pytestmark = [
-    pytest.mark.topology('t0', 't1', 't2', 'm0', 'mx', 't1-multi-asic'),
+    pytest.mark.topology('t0', 't1', 't2', 'm0', 'mx', 't1-multi-asic'), # M1/M2/M3: No need. Link local test on eth0. M0/Mx: ?
```

Comment:
the test case itself requires an update to decouple with eth0

Reply:
I have updated the mark to cover M1/M2/M3. We can have another PR to improve the testcase.